common/astyanax/src/main/java/com/bazaarvoice/emodb/common/cassandra/CassandraConfiguration.java

```diff
@@ -342,6 +343,7 @@ public CqlCluster cluster() {
                     .withClusterName(_cluster)
                     .withLoadBalancingPolicy(_loadBalancingPolicy)
                     .withRetryPolicy(_retryPolicy)
+                    .withoutJMXReporting()
```

Comment:
behaviour has been changed, don't we need that anymore?

Reply:
This was replaced with a custom JMX reporter in `CqlCluster.java`. Additionally, I'm not sure we have ever used it. I just left it here for backward compatibility just in case

Reply:
BTW, I can't view the link you posted, It just takes me to the page of the entire diff.

Reply:
The JMXReporter must not be started until after the cluster has started, or there will be an NPE thrown. This is what led me to relocate this functionality

Reply:
sorry, link was https://github.com/bazaarvoice/emodb/blob/529b3fa9f1437af4afa6495a10a78094c9c8e14d/common/astyanax/src/main/java/com/bazaarvoice/emodb/common/cassandra/CqlCluster.java#L58

Reply:
See comments above

Reply:
we can pass reporter in constructor, but start in `CQLCluster` class

Reply:
The issue is not access to the data, but rather compatibility with the latest version of Dropwizard Metrics. https://docs.datastax.com/en/developer/java-driver-dse/1.6/manual/metrics/

Reply:
missed that clusters are different, marking conversation as resolved